rosetta/docs/NATIVE_FP8.md

```diff
@@ -113,6 +113,7 @@ In addition to the suggested XLA flags mentioned in [this section](https://githu
 export XLA_FLAGS=" \
     --xla_gpu_enable_reduction_epilogue_fusion=false \
     --xla_gpu_enable_triton_gemm=false \
+    --xla_gpu_enable_cudnn_layer_norm=true \
```

Comment:
Should we add them to the container too?

Reply:
I don't think we need. The TE uses its own custom call to the layer norm function, right?

Reply:
Maybe specify in the text bellow that some only help without TE or something like that?

Reply:
I believe whether you set this flag or not, it shouldn't impact the TE because TE replaces the entire transformer layer regardless. This document specifically addresses native FP8 support (i.e., when ENABLE_TE=0). For a concise summary of recommended TE XLA flags, I think people should refer to `rosetta/rosetta/projects/pax/README.md`. That was why we moved out this guide from there.

Reply:
SG.